### ISSUE TITLE
Discrepancy Between Reproduced SFT Model Performance and Released Model

### ISSUE THREAD

**Issue by ByZ0e** [Embodied-Reasoning-Agent/Embodied-Reasoning-Agent#8]
I have attempted to reproduce the SFT training stage following the procedures described in the paper, but the performance of my trained model shows a significant gap compared to the released model. I am writing to confirm if my training setup is correct and ask about potential hyperparameter impacts.

Reproduction Steps: 
First, I trained the model for 1 epoch using the dataset eb_alfred_environment_anchored_prior_dataset.json.
Then, I took the checkpoint from the first stage and continued training for another 1 epoch with the dataset eb_alfred_trajectory_augmented_prior_dataset.json.
All training hyperparameters were kept consistent with the provided train.sh script:
E.g.,
Batch size: 8
Learning rate: 1e-5

Questions
1) Is the two-stage training pipeline (1 epoch on environment-anchored data → 1 epoch on trajectory-augmented data) correct as intended? Not use the two json files at the same time like:
datasets:
  - json_path: xxx/eb_alfred_environment_anchored_prior_dataset.json
    images_folder: ""
    sampling_strategy: "all"
  - json_path: xxx/eb_alfred_trajectory_augmented_prior_dataset.json
    images_folder: ""
    sampling_strategy: "all"
2) How significant are the impacts of adjusting hyperparameters like batch size and min_pixels/max_pixels(I use 448 not 500 instead) on the final model performance?
3) Are there any other potential factors (e.g., data preprocessing details, optimizer settings, training stability tricks) that might cause the performance gap that I may have overlooked?

Any explanations or suggestions would be greatly appreciated. Thank you for your time and effort!
